### PR TITLE
Add open peer review course #186

### LINKED ISSUE #186: Missing course 09 Open Peer Review
Write a course on newly implemented open review process on the OEP and publish it in the academy.

### PR DESCRIPTION
**Pull request by Ludee**
### Added

- Add new open peer review course [(#196)](https://github.com/OpenEnergyPlatform/academy/pull/196)

### CODE REVIEW

File: docs/courses/09_peer_review.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+
+## How to evaluate and present the review? What are badges?
+
+Based on a scientific survey, the individual metadata keys were evaluated and 
+categorized into groups. By accepting all entries in this section, 
+a corresponding badge is awarded. These badges are named after precious metals 
+such as "iron," "bronze," "silver," "gold," and "platinum."
+
+Todo: Link to documentation of the badges
```
Comment (christian-rli) [inline docs/courses/09_peer_review.md]: This TODO is still open

File: docs/courses/09_peer_review.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+### Editor / Community Manager
+
+The editor asks members to start and submit a review for specific tables.
+When a review started and is not completed the editor can close the review to proceed.
+In case the contributor and reviewer do not come an agreement, the editor decides.
+The editor is not implemented yet.
+
+## Supplementary material and further readings
+
+- Stephan's notes and thesis
```
Comment (christian-rli) [inline docs/courses/09_peer_review.md]: Either add the mentioned notes or remove this line.

**Review by christian-rli — CHANGES_REQUESTED** [review]
Content and language of what's there is acceptable :) The course could benefit from one or more screenshots, once possible. If the development notes are removed from the file, it may go public as a first version.

**Comment by Ludee** [comment]
I finished the two open tasks. Please review again!

**Review by christian-rli — APPROVED** [review]

**Pull request merged**